### PR TITLE
Data csv export refactor

### PR DESCRIPTION
**Pull request by minotogna**

### CODE REVIEW

File: server/modules/surveyRdb/service/surveyRdbService.js [hunk 1]
```diff
@@ -1,125 +1,28 @@
...
-const DataTable = require('../schemaRdb/dataTable')
 
 const SurveyManager = require('../../survey/manager/surveyManager')
 const SurveyRdbManager = require('../manager/surveyRdbManager')
-const RecordManager = require('../../record/manager/recordManager')
 
-const _getQueryData = async (surveyId, cycle, nodeDefUuidTable, nodeDefUuidCols = []) => {
-  const surveySummary = await SurveyManager.fetchSurveyById(surveyId, true)
-  const surveyInfo = Survey.getSurveyInfo(surveySummary)
+const _getSurvey = async (surveyId, cycle) => {
```
Comment (fefrain) [inline server/modules/surveyRdb/service/surveyRdbService.js]: it's more a "fetch" than a get...

File: core/stringUtils.js [hunk 2]
```diff
@@ -33,6 +32,14 @@ const normalizeName = R.pipe(
...
 const capitalizeFirstLetter = text => text.charAt(0).toUpperCase() + text.slice(1)
 
+const removeNewLines = R.when(
+  isString,
+  R.pipe(
+    R.split(/\r\n|\r|\n/g),
+    R.join(' ')
+  )
+)
+
```
Comment (fefrain) [inline core/stringUtils.js]: should it be just a string replace instead?

Comment (minotogna) [reply core/stringUtils.js]: I don't think so. It's really a remove new lines. I added a space for making the value more readable

File: server/modules/surveyRdb/service/surveyRdbService.js (hunk 1 above)
Comment (minotogna) [reply server/modules/surveyRdb/service/surveyRdbService.js]: I can rename it after we merge this branch into processing chain poc

**Pull request merged**